src-vue/src/views/ThunderstoreModsView.vue

```diff
@@ -1,19 +1,11 @@
 <template>
-    <div class="fc-container">
+    <div class="fc-container" style="padding: 0">
```

Comment:
I assume the reason the padding is set to zero here instead of for `fc-container` is that this should only apply to this specific div?

Reply:
Exactly